src/main/scala/strawman/collection/immutable/BitSet.scala

```diff
@@ -66,6 +67,13 @@ object BitSet extends SpecificIterableFactory[Int, BitSet] {
       case _          => empty.++(it)
     }
 
+  def empty: BitSet = new BitSet1(0L)
+
+  def newBuilder(): Builder[Int, BitSet] =
+    new ImmutableBuilder[Int, BitSet](empty) {
```

Comment:
This is a performance bug waiting to happen.  Larger immutabe bitsets should absolutely not be built this way; they should be built with a mutable bitset, and upon asking for the result the array should be handed over and/or copied.